Language/Functions/Bits and Bytes/bit.adoc

```diff
@@ -44,6 +44,32 @@ The value of the bit.
 --
 
 [float]
+=== Example Code
+// Describe what the example code is all about and add relevant code   ►►►►► THIS SECTION IS MANDATORY ◄◄◄◄◄
+Prints the output of `bit(n)` for n=0 to n=10 to the serial monitor. Values returned are 1 2 4 8 16 till 1024.
+[source,arduino]
+----
+void setup() {
+Serial.begin(9600); 
+}
+
+void loop() {
+for(int n=0;n<=10;n++)  //loop from n=0 till 10
+  {
+  Serial.print(" ");    //space between numbers, for readability
+  Serial.print(bit(n)); //print the value of bit(n)
+  delay(1000);
+  }
+}
+----
+[%hardbreaks]
+
+[float]
+=== Notes and Warnings
+The variable `n` must be an integer type. float and double will return an error and the program will not compile.
```

Comment:
It seems better to specify the parameter type in the "**Parameters**" section of the reference page.

I'm not sure what the best approach is for defining types of parameters of macros such as `bit`:

https://github.com/arduino/ArduinoCore-API/blob/1.3.1/api/Common.h#L71